AB_Hands_On/AB_Hands_On_FinalSection.ipynb

```diff
@@ -0,0 +1,1571 @@
+{
```

Comment:
<p>This is the proposed new section. Its very rough but should be enough to get an idea of the intent is</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>